lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -98,8 +125,21 @@ sub _hmt_csv {
     while (my $row = $csv->getline($fh) or not $csv->eof()) {
         ($row->[23] and $row->[23] eq "Individual") or next;
         my $name = _process_name @{$row}[0 .. 5];
+        
         next if $name =~ /^\s*$/;
-        push @names, $name;
+        
+        my $date_of_birth = @{$row}[7];
```

Comment:
```suggestion
        my $date_of_birth = $row->[7];
```

Reply:
Perhaps it's also a good idea to make the magic numbers here and on line 126, 127 constants?